p2p/README.md

```diff
@@ -0,0 +1,8 @@
+# Peer-to-Peer Communication
+
+**Important:** The official documentation and specification of the p2p
+communication layer is located in the main CometBFT repository:
+https://github.com/cometbft/cometbft/tree/main/spec/p2p
+
+This repository stores unfinished work, work in progress, and auxiliary
+material for the official specification of the p2p layer.
```

Comment:
Should we add a small table of contents here?

Did you record somewhere that in `p2p/diagrams/pex-reactor.png` there are two different protocols, PEX on the left and connection management on the right?

Reply:
Table of contents: https://github.com/cometbft/knowledge-base/pull/12/commits/f96e89501e7b5a7beb33f80e1086aec071e09eed